dev_env/settings.py

```diff
@@ -10,7 +10,7 @@
     SECRET_KEY = 'djangoledger1234!DoNotUse!BadIdea!VeryInsecure!'
 DEBUG = True
 
-ALLOWED_HOSTS = ['127.0.0.1', '192.168.1.102', 'localhost']
+ALLOWED_HOSTS = ['127.0.0.1', '192.168.1.102', 'localhost','0.0.0.0']
```

Comment:
This is a security issue for the user that is running the server since it binds all network interfaces by default.
Can we include additional instructions on the README.md so the user can change the ALLOWED_HOSTS setting as needed?

Reply:
I added instructions related to ALLOWED_HOSTS in readme.md file.